histomics_stream/tensorflow.py

```diff
@@ -302,6 +302,21 @@ def _py_read_chunk_pixels(
         level = level.numpy()
 
         if re.compile(r"\.svs$").search(filename):
+            import large_image
+
+            ts = large_image.open(filename)
+            chunk = ts.getRegion(
+                scale=dict(level=level),
+                format=large_image.constants.TILE_FORMAT_NUMPY,
+                region=dict(
+                    left=chunk_left,
+                    top=chunk_top,
+                    width=chunk_right - chunk_left,
```

Comment:
As a simplification, you can pass `right` and `bottom` to `getRegion` as part of the `region` dict rather than `width` and `height`.